### PR TITLE
Use multiprocessing in LocalhostEnvironment

### PR DESCRIPTION
**Pull request by Wiezzel**
Running GRPC server in a subprocess instead of a thread provides a great speed up and allows to implement proper stop().

### CODE REVIEW

**Review by Krigpl — APPROVED** [review]
thumbupglitch

File: requirements.txt [hunk 1]
```diff
@@ -24,6 +24,7 @@ crossbar==19.6.2
 cryptography==2.7
 cytoolz==0.9.0.1
 dataclasses==0.6
+dill==0.3.0
```
Comment (Krigpl) [inline requirements.txt]: That looks like it belongs in `requirements-test`

Comment (Wiezzel) [reply requirements.txt]: Meh, putting it in `requirements-test.txt` breaks buildbot. 😢 
```
E: tests/golem/envs/localhost.py:8:0: Unable to import 'dill' (import-error)\n
```

Comment (Wiezzel) [reply requirements.txt]: Maybe it should be in `requirements-lint.txt` as well?

Comment (Krigpl) [reply requirements.txt]: lint has `-r requirements-test.txt` in it

Comment (Krigpl) [reply requirements.txt]: Not sure if that works tho

Comment (Wiezzel) [reply requirements.txt]: I'll try.

Comment (Wiezzel) [reply requirements.txt]: Adding to `requirements-lint.txt` helped.

**Pull request merged**